components/stack_exchange/stack_exchange.app.js

```diff
@@ -35,6 +36,61 @@ module.exports = {
       label: "Keywords",
       description: "Keywords to search for in questions",
     },
+    questionIds: {
+      type: "string[]",
+      label: "Questions",
+      description: "Questions to monitor (max: 100)",
+      useQuery: true,
+      async options(context) {
+        const q = context.query || '';
```

Comment:
Any specific reason you need to default to the empty string here, or will it work by just letting `context.query` be undefined when it's not present?

Reply:
Yeah, the StackExchange API doesn't work well when we set `q` to `null`, which is the value that `context.query` has when it's not set. I spend a couple of hours scratching my head because of this :sob:

Reply:
Answered before, but just for future contributors:
`context.query` is `null` when left unset, which doesn't go well with the StackExchange API.